routes/productRoute.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+// eslint-disable-next-line new-cap
+const route = express.Router();
+const productController = require('../controllers/productController');
+
+route.post('/add', productController.addproduct);
```

Comment:
As RestAPIs official documentation (https://restfulapi.net/rest-api-design-tutorial-with-example/ ) all URLs should look the same for DELETE,GET,POST or PUT so the URL for adding product should be ( / ) not ( /product )